vmware/ngc/NGC-AdapterManager/src/main/resources/constants.properties

```diff
@@ -1,6 +1,6 @@
 OPENSDS_TENANT=admin
```

Comment:
@NajmudheenCT @kumarashit I think, we should consider to avoid 'OpenSDS' name based variables and configurations. For the new designs, please start this point. Old ones, what can we do?